src/test/kotlin/dev/fakek/fakes/FakeBooleanTest.kt

```diff
@@ -0,0 +1,24 @@
+package dev.fakek.fakes
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Test
+import strikt.api.expectThat
+import strikt.assertions.isEqualTo
+
+internal class FakeBooleanTest {
+
+    private val trueBoolean by lazy { true }
+    private val fakerBoolean by lazy {
+        mockk<FakerBoolean>()
+            .also { every { it.bool() } returns trueBoolean }
```

Comment:
With MockK defining the type instead of relying on inference can actually clean things up a bit. Another route for defining mocks is:

```kotlin
private val fakerBoolean: FakerBoolean by lazy {
        mockk { 
            every { it.bool() } returns trueBoolean 
        }
}

```

I'm not too concerned about this but it's something I'm thinking about including in a style guide once I get around to creating it. I'd be interested in your thoughts on that.